docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+LemonBoy’s bar
+=============
+
+.. note:: There is currently only a script to run inside i3, but you can just strip the
+          i3-specific parts of the binding.
```

Comment:
This means that you must have `python powerline-bar.py --i3` and must not import or use `i3` without this switch.